translationCoreApps/VerseCheck
# Fix issue with loading VerseCheck on build, not issue on nom start

## This PR addresses:
Build not loading tools because case sensitive load affects builds but not develop, npm start.

## To Test:
Create build and see if it works.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/versecheck/27)
